src/Modules/Database/Database.ts

```diff
@@ -0,0 +1,111 @@
+import { booster } from '@booster-ts/core';
+import inject from '../../injector';
+import { Client, QueryResult } from 'pg';
+import { ErrorModule } from '../Error/Error';
+import { ConfigModule } from '../Config/Config';
+import SimpleSql from '@everydayheroes/simplesql';
+
+/**
+ * DatabaseModule
+ * @description Newly Generated Module
+ * @author ImOverlord
+ */
+@booster({
+    type: "Module"
+})
+export class DatabaseModule {
+
+    /** pg Client */
+    private client: Client;
+    /** SQL Engine */
+    private engine: SimpleSql;
+
+    constructor(
+        private error: ErrorModule,
+        private config: ConfigModule
+    ) {
+        this.engine = new SimpleSql('pgEngine');
+        this.client = new Client({
+            connectionString: this.config.getConfig().DB
+        });
+    }
+
+    /**
+     * init
+     * @description Init Database
+     */
+    public init(): Promise<any> {
+        return this.client.connect()
+        .then(() => {
+            return this.client.query("SET SCHEMA 'public'");
+        });
+    }
+
+    /**
+     * query
+     * @description Query Database
+     * @param command to send
+     * @param args in command
+     */
+    public query(command: string, args?: Array<any>): Promise<QueryResult> {
```

Comment:
Maybe we could use the keywords async/await, to make the code more readable, I converted this function to async/await as example
public async query2(command: string, args?: Array<any>): Promise<QueryResult> {  
        if (args === undefined)  
            args = [];  
        try {  
            return await this.client.query(command, args)  
        } catch (error) {  
            this.error.createCustomError('Database', "Query Failed", error)  
        }  
    }